tests/CMakeLists.txt

```diff
@@ -46,6 +46,12 @@ endfunction()
 add_subdirectory(fixtures)
 add_subdirectory(unit)
 
+#
+# Additional external libraries
+#
+include(Boost_DD)
+include(Json_DD)
+
```

Comment:
Doesn't this automatically pick up the parent includes?

Reply:
find_package does not add packages globally I think, while other methods do.

There's a GLOBAL flag that's available in 3.24 that I would like to use though and get rid of this in the future...

Reply:
In Sunshine, I just bumped one of our docker images from 3.18 to a the latest cmake, so it shouldn't be too hard to do that if there's a few others lagging.

Reply:
Does it mean I should bump CMake to 3.24 now and remove these?

Reply:
You could if you'd like to